Userland/Libraries/LibGUI/Event.h

```diff
@@ -318,6 +318,12 @@ enum MouseButton : u8 {
     Forward = 16,
 };
 
+enum WheelDirection : u8 {
+    WheelNone = 0,
+    WheelVertical = 1,
+    WheelHorizontal = 2,
+};
```

Comment:
Why did you choose a plain enum here? After all, in the window server there's an enum class...

Reply:
To be honest I've just mimicked mouse button enum that's defined in the same fashion above this one